src/ripple/basics/base_uint.h

```diff
@@ -143,7 +143,7 @@ class base_uint
         assert (vch.size () == size ());
 
         if (vch.size () == size ())
```

Comment:
Since we already assert this condition right above, I'm just going to collapse this conditional in my patch later today.

Reply:
I wouldn't remove the condition. Although it would be a logic error is the sizes don't match, it's also a change in behavior that would fall outside the scope of this PR. We do sometimes do small unrelated cleanups in PRs, but base_uint is a pretty fundamental type and changes in behavior need to be carefully considered.

So good idea, but my vote is "not for this PR".

Reply:
Very valid points. I'll save it for a different PR. Unnecessary coupling isn't good given how new I am to the codebase.